std/file.d

```diff
@@ -5098,13 +5098,15 @@ slurp(Types...)(string filename, scope const(char)[] format)
     import std.exception : enforce;
     import std.format : formattedRead;
     import std.stdio : File;
+    import std.string : stripRight;
 
     auto app = appender!(typeof(return))();
     ElementType!(typeof(return)) toAdd;
     auto f = File(filename);
     scope(exit) f.close();
     foreach (line; f.byLine())
     {
+        line = stripRight(line, ['\r']);
```

Comment:
`['\r']` doesn't compile in `@nogc`. Maybe just use `"r"`?

Reply:
Yes, fortunately the right overload exists.